slick/src/main/scala/slick/jdbc/MySQLProfile.scala

```diff
@@ -319,6 +319,11 @@ trait MySQLProfile extends JdbcProfile { profile =>
   }
 
   class JdbcTypes extends super.JdbcTypes {
+    override val stringJdbcType = new StringJdbcType
+    override val uuidJdbcType = new UUIDJdbcType
+    override val instantType : InstantJdbcType = new InstantJdbcType
+    override val localDateType : LocalDateJdbcType = new LocalDateJdbcType
+    override val localDateTimeType : LocalDateTimeJdbcType = new LocalDateTimeJdbcType
```

Comment:
Isn't creating named classes just noise here? It seems they are used once, so previous approach of anonymous classes would work fine.

Reply:
I understood, but I don't think it's noise.
For me, it's more concise to use each implementation as class and use hierarchy to reuse methods - like we do with `MySQLTimeJdbcType` trait.